### DISCUSSION TITLE
0.3.0

### DISCUSSION THREAD

**Discussion by mgmeyers** [mgmeyers/obsidian-kanban#89] [Releases]
- Adds time picker and time support
  - Note: by default, the time picker can be triggered with `@@`
  - Note: times require dates
- Adds option to add date/time to archived cards
- Adds command to archive all completed cards on the board
- Adds command to convert empty note to a Kanban board
- Removes cmd/ctrl + click in favor of double clicking to edit cards and lane titles
- Ensures new cards and Kanban boards are created in the correct folder

<img width="1064" alt="Screen Shot 2021-05-14 at 12 24 09 PM" src="https://user-images.githubusercontent.com/2694747/118319385-8941cf00-b4af-11eb-9669-26a312cfcd4e.png">

<img width="289" alt="Screen Shot 2021-05-14 at 12 25 27 PM" src="https://user-images.githubusercontent.com/2694747/118319396-8e068300-b4af-11eb-9d70-ecd952f09072.png">

<hr /><em>This discussion was created from the release <a href='https://github.com/mgmeyers/obsidian-kanban/releases/tag/0.3.0'>0.3.0</a>.</em>

**Comment by ryanjamurphy** [comment]
> Removes cmd/ctrl + click in favor of double clicking to edit cards and lane titles

My hero
